database/ddl.sql

```diff
@@ -0,0 +1,27 @@
+create table if not exists countries
+(
+    id bigserial not null
+        constraint countries_pk
+        primary key,
+    name varchar,
+    world_region varchar,
+    government varchar,
```

Comment:
It is better to rename column to government_type

Reply:
done